packages/cli-kit/src/public/node/global-context.ts

```diff
@@ -0,0 +1,36 @@
+export interface GlobalContext {
+  currentCommandId: string
+  [key: string]: string
```

Comment:
What's this for? It basically makes GlobalContext close to untyped -- I haven't seen a use of it. Would drop it if not used.